03-specifying-sample-designs.Rmd

```diff
@@ -210,7 +212,7 @@ In the chunk above, the design object is printed, and the object summary is show
 
 - **Description**: A population is divided into mutually exclusive subpopulations (strata), and then samples are selected independently within each stratum.
 - **Requirements**: The sampling frame must include the information to divide the population into groups for every unit.
-- **Advantages**: This design ensures sample representation in all subpopulations. Often, for the same sample size as a SRS sample, the standard errors are smaller, so it is a more efficient design. This is true if the strata are correlated with outcomes.
+- **Advantages**: This design ensures sample representation in all subpopulations. Often, for the same sample size as a SRS sample, the standard errors are smaller, so it is a more efficient design. This is true if the strata are correlated with outcomes. 
```

Comment:
"This is true if the strata are correlated with outcomes." - Should we add why?

Reply:
Agreed.  I also restructured the sentence as it was a bit awkward to me upon first read.